backend/app/tests.py

```diff
@@ -5,149 +5,11 @@
-
-    def test_get_count_of_word(self):
-        doc = Document.objects.get(title='doc3')
-        self.assertEqual(doc.get_count_of_word('do'), 2)
-        self.assertEqual(doc.get_count_of_word('ThisWordIsNotThere'), 0)
-
-    def test_get_count_of_words(self):
-        doc = Document.objects.get(title='doc3')
-        self.assertEqual(doc.get_count_of_words(['do', 'as', 'you']), Counter({'do': 2, 'as': 2, 'you': 1}))
-        self.assertEqual(doc.get_count_of_words(['ThisWordIsNotThere', 'well']),
-                         Counter({'ThisWordIsNotThere': 0, 'well': 0}))
-
-    def test_words_associated(self):
-        doc = Document.objects.get(title='doc6')
-        self.assertEqual(doc.words_associated('his'), Counter({'cigarette': 1, 'speech': 1}))
-
-    def test_get_word_windows(self):
-        doc = Document.objects.get(title='doc6')
-        windows_1 = Counter(
-            {'he': 1, 'lit': 1, 'cigarette': 1, 'and': 1, 'then': 1, 'began': 1, 'speech': 1, 'which': 1})
-        windows_2 = Counter({'her': 2, 'of': 1, 'and': 1, 'handed': 1, 'proposal': 1, 'drowned': 1, 'the': 1})
-        self.assertEqual(doc.get_word_windows('his', window_size=2), windows_1)
-        self.assertEqual(doc.get_word_windows(['purse', 'tears']), windows_2)
-
-    def test_get_word_freq(self):
-        doc = Document.objects.get(title='doc7')
-        self.assertEqual(doc.get_word_freq('sad'), 0.13333333333333333)
-
-    def test_get_word_frequencies(self):
-        doc = Document.objects.get(title='doc8')
-        word_freqs = {'peace': 0.02702702702702703, 'died': 0.02702702702702703, 'foobar': 0.0}
-        self.assertEqual(doc.get_word_freqs(['peace', 'died', 'foobar']), word_freqs)
-
-    def test_get_part_of_speech_tags(self):
-        doc = Document.objects.get(title='doc9')
-        tags_1 = [('They', 'PRP'), ('refuse', 'VBP'), ('to', 'TO'), ('permit', 'VB')]
-        tags_2 = [('the', 'DT'), ('refuse', 'NN'), ('permit', 'NN'), ('.', '.')]
-        self.assertEqual(doc.get_part_of_speech_tags()[:4], tags_1)
-        self.assertEqual(doc.get_part_of_speech_tags()[-4:], tags_2)
-
-    def test_part_of_speech_words(self):
-        doc = Document.objects.get(title='doc8')
-        words = {'JJ': Counter({'beautiful': 3}), 'VBD': Counter({'died': 1}), 'NN': Counter({'peace': 1})}
-        self.assertEqual(doc.get_part_of_speech_words(['peace', 'died', 'beautiful', 'foobar']), words)
-
-    def test_update_metadata(self):
-        doc = Document.objects.get(title='doc1')
-        doc.update_metadata({'date': '1903'})
-        new_attribute = {'cookies': 'chocolate chip'}
-        doc.update_metadata(new_attribute)
-        new_attribute_2 = {'text': 'The quick brown fox jumped over the lazy dog.'}
-        doc.update_metadata(new_attribute_2)
-        self.assertEqual(doc.date, 1903)
-        self.assertEqual(doc.new_attributes['cookies'], 'chocolate chip')
-        self.assertEqual(doc.word_count, 9)
```

Comment:
I honestly have no idea how this snuck into the boilerplate template. Good catch, thanks for cleaning it up.